.travis.yml

```diff
@@ -1,23 +1,23 @@
+version: ~> 1.0
 language: julia
 os:
     - linux
     - osx
 julia:
-    - 0.7
-    - '1.0'
+    - 1.0
+    - 1.4
     - nightly
 notifications:
     email: false
 matrix:
     allow_failures:
-    - julia: '1.0'
     - julia: nightly
 git:
     depth: 99999
 env:
     global:
     - PYTHON=conda
-before_install:
+#before_install:
```

Comment:
If you like we can delete all of these commented out lines :)

Reply:
Ok